content/research_report/introductions.md

```diff
@@ -42,20 +39,17 @@ and an optimal approach often involves combining a few classes of regressors des
 
 ## Implementation of denoising step
 
-<!-- How denoising is traditionally done in propriatory software -->
-<!-- NC: changed this to present tense as I assume people still use the software being described? -->
+<!-- aim of the paragraph: How denoising is traditionally done in propriatory software -->
 Which approach to adopt is straightforward when users use preprocessing software with statistical modelling functionality.
 Each software implements their own preferred strategy,
 for instance, FSL adds the motion parameters by default, and provides an option to run ICA-AROMA.
 The noise regressors are preselected,
 and thus user intervention is avoided.
 However, when users wish to use additional regressors, the process can be complicated.
-With the recent benchmark papers on confound denoising, <!-- NC: references here?  -->
+With the recent benchmark papers on confound denoising highlighting methods such as ICA-AROMA and CompCor {cite:p}`ciric_benchmarking_2017,parkes_evaluation_2018`, 
```

Comment:
I make this part more precise about the two class of methods was thinking about writing this sentence. Does this make sense?

Reply:
yes, perfect